code/swb_base/AttachmentBase.cs

```diff
@@ -0,0 +1,170 @@
+using System;
```

Comment:
Do we really wanna support attachments in vanilla?

Reply:
No, I can remove it, I wonder how much other code will change though.

Reply:
Yeah, it's a bit too intertwined with the base code, it would have to stay.